### PR TITLE
Header update

### LINKED ISSUE #123: website header update
有些desription&url是舊的
- [x] description
- [x] canonical url 
- [x] title

### PR DESCRIPTION
**Pull request by barry800414**
to fix #123

### CODE REVIEW

File: about_us.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
     <title>工時透明化運動 - 最新資料</title>
```
Comment (mark86092) [inline about_us.html]: Title, thanks

Comment (barry800414) [inline about_us.html]: 你超快XD  我才想說要檢查

Comment (mark86092) [inline about_us.html]: 其實我還沒看完xd

**Comment by mark86092** [comment]
I reviewed it, many thanks

**Pull request merged**